spec/system/logins_spec.rb

```diff
@@ -29,6 +29,30 @@
       expect(user.role).to eq("challenge_manager")
       expect(page).to(be_axe_clean)
     end
+
+    it "manage submissions index page is accessible with no challenges" do
```

Comment:
I think we should break out the main feature areas into separate files. can you create a new one for the manage submission pages? they should still reside in `spec/system`